Scripts/install.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 else
     BUILD_FOLDER=".build/debug"
     swift build --disable-sandbox
-    MAJOR_VERSION=$(swift --version | awk '{for(i=1;i<=NF;i++){if($i ~ /^[0-9]+\.[0-9]+$/){print $i; break}}}' | cut -d '.' -f 1)
+    MAJOR_VERSION=$(swift --version | awk '{for(i=1;i<=NF;i++){if($i ~ /^[0-9]+\.[0-9]+\.[0-9]?$/){print $i; break}}}' | cut -d '.' -f 1)
```

Comment:
I'm not a regex expert, but shouldn't this be (\.[0-9]+)? To have all the group optional and support more than one digit?

Reply:
Certainly, thank you.

Reply:
fixed!